### DISCUSSION TITLE
Welcome to thinkscript Discussions!

### DISCUSSION THREAD

**Discussion by 2187Nick** [2187Nick/thinkscript#1] [Announcements]
## 👋 Welcome!
  Use this Discussions as a place to ask questions, request indicators and share ideas:

**Comment by tdao24-a11y** [comment]
Hi Nick
i read and reread, but not really understand your option_volume_and_openInterest_vertical_axis chart work.  would you please explain with more details?  what exactly do i need to look for when open the chart in the morning when market is open to try /es.  please advise and thank you very much in advance.  Vincent.

**Reply by 2187Nick** [reply]
Hello,
First try the standard version and use it with SPY to make sure it works. Then with the /ES version you will need to manually enter the option contract symbol you want to view OI or Volume for. 

Make sure you are following the instructions for chart settings:
Time frame: 1 Day
Aggregation Period: 1 Minute
Time Axis: Expansion Area: 200 bars to the right

If you still have questions you can stop by the discord. Lots of members there who can help troubleshoot.
https://discord.com/invite/vxKepZ6XNC
